module/dkg/broker.go

```diff
@@ -116,18 +120,22 @@ func (b *Broker) Broadcast(data []byte) {
 	}
 	maxedExpRetry := retry.WithMaxRetries(retryMax, expRetry)
 
-	err = retry.Do(context.Background(), maxedExpRetry, func(ctx context.Context) error {
-		err := b.dkgContractClient.Broadcast(bcastMsg)
+	f := func() {
+		err = retry.Do(context.Background(), maxedExpRetry, func(ctx context.Context) error {
+			err := b.dkgContractClient.Broadcast(bcastMsg)
+			if err != nil {
+				b.log.Error().Err(err).Msg("error broadcasting DKG result, retrying")
```

Comment:
```suggestion
				b.log.Warn().Err(err).Msg("error broadcasting DKG result, retrying")
```

Reply:
I think it makes sense to leave this as an error since failing to broadcast has important consequences on the node.

Reply:
On a separate note, the error message is not correct: DKG result is one of the message types that can be broadcast, but the error here is about any message type. 
```suggestion
				b.log.Error().Err(err).Msg("error broadcasting, retrying")
```